packages/lingua-franca/src/components/layout/cookie.scss

```diff
@@ -1,94 +0,0 @@
-@charset "utf-8"; /*! mscc v0.4.2 - Copyright 2020 Microsoft Corporation */
-.cc-banner {
-  position: relative;
-  font-size: 12px;
-}
-.cc-banner .hide {
-  display: none;
-}
-.cc-banner a,
-.cc-banner div,
-.cc-banner span,
-.cc-banner svg {
-  margin: 0;
-  padding: 0;
-  text-decoration: none;
-}
-.cc-banner .cc-v-center {
-  display: inline;
-  vertical-align: middle;
-  line-height: 2em;
-}
-.cc-banner[dir="rtl"] .cc-float-left,
-.cc-float-right {
-  float: right;
-}
-.cc-banner[dir="rtl"] .cc-float-right,
-.cc-float-left {
-  float: left;
-}
-.cc-banner {
-  font-family: "Segoe UI", "Helvetica Neue", Helvetica, Arial, sans-serif;
-  color: #231f20;
-  background: #f2f2f2;
-  display: none;
-  text-align: center;
-  padding: 0 1em;
-  margin: 0;
-}
-@media (min-width: 768px) {
```

Comment:
I think it was the style file for a banner that asks users if they want to accept cookies. I deleted it because I don't think we need that banner, because AFAIK we are not using cookies.